### DISCUSSION TITLE
Setting up Cypress with axe for accessibility

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#341] [Announcements]
# Setting up Cypress with axe for accessibility

Having an accessible website is important. With the combination of Cypress and axe you can verify that your website is accessible, with only 2 extra lines of code.

https://timdeschryver.dev/blog/setting-up-cypress-with-axe-for-accessibility

**Comment by HappyMan0001** [comment]
Hi, I saw this blog. It has helped. I have one question for you: 
If I want to only get the failures for certain Standards (ie: "wcag2a", "wcag412") how do I create that rule. I been trying to use a seperate .ts doc (axe-configuration.ts) inside of it, I'm doing this: 
export const defaultAxeConfiguration = 
{
    //branding: String,
    //reporter: 'no-passes',
    //checks: [{id: '[data-testid="add-new-briefing"]'}],
    rules: [
        {
            id: '[id="storymaps-root"]',
            matches: [
            "wcag2a",
            "wcag412"]
        }
    ],
    //standards: Object,
    //locale: Object,
    //axeVersion: String,
    disableOtherRules: true,
    //noHtml: Boolean
}

I tried various things so I may only get results for "wcag2a", "wcag412". Any help is appreciated.

**Reply by HappyMan0001** [reply]
Ok, I figured it out. It's best to use: cy.checkA11y(null, null, null) <-- you may remove the nulls and input whatever is required: 
cy.checkA11y(null, {runOnly: {type: 'tags', values: ['wcag2a', 'wcag412']}},  cy.AccessibilityViolations)
So on this I left the first parameterer as null, the second one only allows the types of tags and the third one is a written cypress function on the commands page, that I wrote. 
thanks for not replying. I hope this helps others.

**Reply by timdeschryver** [reply]
Thanks for getting back to this with an answer @HappyMan0001
